ipyopenlayers/example.py

```diff
@@ -24,28 +24,62 @@ class TileLayer(Widget):
     _view_module_version = Unicode(module_version).tag(sync=True)
     url = Unicode('https://{a-c}.tile.openstreetmap.org/{z}/{x}/{y}.png').tag(sync=True)
 
-class BaseOverlay(DOMWidget):
-   _model_name = Unicode('BaseOverlayModel').tag(sync=True)
+class BaseOverlay(DOMWidget): 
+   
    _model_module = Unicode(module_name).tag(sync=True)
    _model_module_version = Unicode(module_version).tag(sync=True)
-   _view_name = Unicode('BaseOverlayView').tag(sync=True)
    _view_module = Unicode(module_name).tag(sync=True)
    _view_module_version = Unicode(module_version).tag(sync=True)
-   overlay_type = Unicode().tag(sync=True)
    position = List([0, 0]).tag(sync=True)
 
 class ImageOverlay (BaseOverlay):
-   overlay_type = Unicode('image').tag(sync=True)
+   _view_name = Unicode('ImageOverlayView').tag(sync=True)
+   _model_name = Unicode('ImageOverlayModel').tag(sync=True)
+
    image_url = Unicode('').tag(sync=True)
 
 class VideoOverlay (BaseOverlay):
-   overlay_type = Unicode('video').tag(sync=True)
+   _view_name = Unicode('VideoOverlayView').tag(sync=True)
+   _model_name = Unicode('VideoOverlayModel').tag(sync=True)
+   
    video_url = Unicode('').tag(sync=True)
 
 class PopupOverlay (BaseOverlay):
-   overlay_type = Unicode('popup').tag(sync=True)
+   _view_name = Unicode('PopupOverlayView').tag(sync=True)
+   _model_name = Unicode('PopupOverlayModel').tag(sync=True)
+   
    popup_content = Unicode('').tag(sync=True)
 
+class BaseControl(DOMWidget):
+   _model_name = Unicode('BaseControlModel').tag(sync=True)
+   _model_module = Unicode(module_name).tag(sync=True)
+   _model_module_version = Unicode(module_version).tag(sync=True)
+   _view_name = Unicode('BaseControlView').tag(sync=True)
+   _view_module = Unicode(module_name).tag(sync=True)
+   _view_module_version = Unicode(module_version).tag(sync=True)
```

Comment:
We probably don't need `_model_name` and `_view_name` here

```suggestion
   _model_module = Unicode(module_name).tag(sync=True)
   _model_module_version = Unicode(module_version).tag(sync=True)
   _view_module = Unicode(module_name).tag(sync=True)
   _view_module_version = Unicode(module_version).tag(sync=True)
```